columnflow/production/normalization.py

```diff
@@ -98,17 +98,22 @@ def get_br_from_inclusive_dataset(
             # skip processes that are not covered by any dataset or irrelevant for the used dataset
             # (identified as leaf processes that have no occurrences in the stats)
             # (or as non-leaf processes that are not in the stitching datasets)
-            is_leaf = child_proc.is_leaf_process
+            is_leaf = str(child_proc.id) in sum_mc_weight_per_process.keys()
```

Comment:
```suggestion
            is_leaf = child_proc.is_leaf_process
            child_in_weight_procs = str(child_proc.id) in sum_mc_weight_per_process
```
I would not change the definition of is_leaf, since we need it in the if statement in line 102, but I would add a new variable